content/posts/redis-modules-into-arm-land-part1.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Getting Redis Modules into ARM land - Part 1"
+date: 2019-11-13
+authors:
+  - Rafi Einstein
+tags: ["redisedge", "arm"]
+---
+
+Our motivation at Redis Labs of getting into ARM land and dragging Redis Modules along was RedisEdge.
+Redis, of course, has long been native in this land, in both glibc and alpine/musl variants.
+Redis Modules, although already on the multi-platform scene (running on various Linux distributions and supporting macOS, mainly for the sake of development experience), have been more of an enterprise/data-center thing, all until RedisEdge came along, targeting IoT devices.
+In this series of posts, I'll describe our vision of ARM platform support and the developer user experience, as well as the steps we took along the path, as the way was as valuable as the outcome.
+
+If you'll follow through, you'll end up with a fully functional ARM build laboratory.
+
+## Inside RedisEdge
+
+![redisedge1](/redis-modules-arm-1.png)

```

Comment:
Source image looks "fuzzy" to me

Reply:
Got it from here: https://redislabs.com/redis-enterprise/redis-edge/